src/ts/services/keyring-vault-proxy.ts

```diff
@@ -167,3 +169,10 @@ export function importAccountFromJson (json: KeyringPair$Json, password?: string
     })
   })
 }
+
+export function decodeAddress (key: string | Uint8Array, ignoreChecksum?: boolean):
+  Promise<Uint8Array> {
+  return cryptoWaitReady().then(() => {
+    return new Keyring({ type: 'sr25519' }).decodeAddress(key, ignoreChecksum)
```

Comment:
should we check `ignoreChecksum ` before passing it to decodeAddress since it is optional

Reply:
According to Polkadot's doc, it is temp solution. It'll be probably removed in future releases. Let's keep it a while and revisit it. Added TODO